geoapi/tasks/external_data.py

```diff
@@ -375,90 +375,141 @@ def import_files_recursively_from_path(session, tenant_id: str, userId: int, sys
                     raise
 
 
+def _get_user_with_valid_token(project):
+    """ Return a user with valid token
+
+        Returns None if no such user exists.
+    """
+    importing_user = next(
+        (u for u in project.users if u.has_unexpired_refresh_token() or u.has_valid_token()), None)
+    return importing_user
+
+
 @app.task()
-def refresh_observable_projects():
+def refresh_watch_content_projects():
```

Comment:
Pedantic comment #3: the naming of this feels awkward, what do you think about a minor tweak to `refresh_projects_watch_content()`?

Reply:
thanks! will switch.